testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/adapter/example/fuse/EAP6Fuse6HawtioAdapterTest.java

```diff
@@ -112,7 +112,7 @@ public void hawtioLoginAndLogoutTest() {
         WaitUtils.waitForPageToLoad();
 
         log.debug("log in");
-        testRealmLoginPageFuse.assertCurrent();
+        waitUntilUrlStartsWith(testRealmLoginPageFuse.toString(), 60);
```

Comment:
I'd suggest the same approach used in FuseAdapterTest 

- https://github.com/keycloak/keycloak/commit/ac6d877954a01ad77de8ae7826fa398612eadd96#diff-0811a74e7c159fc73e1604954ca77456R89
- https://github.com/keycloak/keycloak/commit/ac6d877954a01ad77de8ae7826fa398612eadd96#diff-0811a74e7c159fc73e1604954ca77456R137
- https://github.com/keycloak/keycloak/commit/ac6d877954a01ad77de8ae7826fa398612eadd96#diff-0811a74e7c159fc73e1604954ca77456R119